tools/mathjax-test-context.js

```diff
@@ -1,4 +1,6 @@
-document.documentElement.dataset.mathjaxContext = JSON.stringify({
+const disabled = window.location.search.indexOf('latex=false') !== -1;
+
+document.getElementsByTagName('html')[0].dataset.mathjaxContext = JSON.stringify({
 	outputScale: 1.1,
-	renderLatex: true
+	renderLatex: !disabled
```

Comment:
Will be `true` unless explicitly disabled. This seems like a better state for most tests and demo pages using HTML block.